### PR TITLE
Updated and made consistent.

### PR DESCRIPTION
**Pull request by hannanelson**
PDF
[Sending API Analytics Event Data from Runtime Manager to External Databases.pdf](https://github.com/mulesoft/docs-runtime-manager/files/4714179/Sending.API.Analytics.Event.Data.from.Runtime.Manager.to.External.Databases.pdf)

### CODE REVIEW

File: modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc [hunk 1]
```diff
@@ -284,31 +285,31 @@ CREATE TABLE MULE_EVENTS_BUSINESS (
...
+----
+--
 
 == See Also
 
-* xref:monitoring.adoc[Monitoring Applications and Servers]
 * xref:sending-data-from-arm-to-external-analytics-software.adoc[Export Data to External Analytics Tools]
 * xref:managing-servers.adoc[Servers, Server Groups, and Clusters]
-* xref:deploying-to-your-own-servers.adoc[Deploying to Your Own Servers]
-* xref:managing-deployed-applications.adoc[Manage Deployed Applications]
```
Comment (marialuzshin) [inline modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc:312]: Are we removing this link for a particular reason?

File: modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc [hunk 2]
```diff
@@ -284,31 +285,31 @@ CREATE TABLE MULE_EVENTS_BUSINESS (
...
     jdbcUrl: jdbc:sqlserver://192.168.61.128:1433;databaseName=Mule;
     user: root
     pass: test
-....
+----
+--
 
 == See Also
 
-* xref:monitoring.adoc[Monitoring Applications and Servers]
```
Comment (marialuzshin) [inline modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc:308]: Are we removing this link for a particular reason?

File: modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc [hunk 3]
```diff
@@ -261,42 +266,38 @@ CREATE TABLE MULE_API_ANALYTICS (
...
     driver: com.microsoft.sqlserver.jdbc.SQLServerDriver
     jdbcUrl: jdbc:sqlserver://192.168.61.128:1433;databaseName=Mule;
     user: root
     pass: test
-....
+----
 
 == See Also
 
-* xref:monitoring.adoc[Monitoring Applications and Servers]
```
Comment (marialuzshin) [inline modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc:296]: Are we removing this link for a particular reason?

**Comment by hannanelson** [comment]
Hi @marialuzshin - I removed the links because I felt they didn't relate to the content - they are just a laundry list of links. 

The topics still exist and can be accessed from the left-hand navigation TOC. If we just add a bunch of cross-references that don't necessarily relate, we make it harder to find the links that do.

But if you think a link is essential to the topic, I can add it back. I'm just trying to streamline.

**Review by marialuzshin — APPROVED** [review]
LGTM!

File: modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc [hunk 4]
```diff
@@ -261,42 +266,38 @@ CREATE TABLE MULE_API_ANALYTICS (
...
 * xref:sending-data-from-arm-to-external-analytics-software.adoc[Export Data to External Analytics Tools]
 * xref:managing-servers.adoc[Servers, Server Groups, and Clusters]
-* xref:deploying-to-your-own-servers.adoc[Deploying to Your Own Servers]
-* xref:managing-deployed-applications.adoc[Manage Deployed Applications]
-* xref:managing-applications-on-your-own-servers.adoc[Managing Applications on Your Own Servers]
-* https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/arm-rest-services[REST API]
+* xref:deploying-to-your-own-servers.adoc[Deploying to Your Servers]
```
Comment (kbostrom) [inline modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc]:
```suggestion
* xref:deploying-to-your-own-servers.adoc[Deploy to Your Servers]
```
...if it isn't too much right now....

File: modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc [hunk 5]
```diff
@@ -261,42 +266,38 @@ CREATE TABLE MULE_API_ANALYTICS (
...
+* xref:managing-applications-on-your-own-servers.adoc[Managing Applications on Your Servers]
```
Comment (kbostrom) [inline modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc]: ditto re. "Manage"

File: modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc [hunk 6]
```diff
@@ -50,66 +50,63 @@ To install the ZIP file:
...
 |String
-|The username to connect to the database server.
+|The username to use to connect to the database server
 |Required
 |
 
 |`pass`
 |String
-|The password to connect to the database server.
+|The password to connect to the database server
```
Comment (kbostrom) [inline modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc:81]:
```suggestion
|The password to use to connect to the database server
```

File: modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc [hunk 7]
```diff
@@ -284,31 +285,31 @@ CREATE TABLE MULE_EVENTS_BUSINESS (
 CREATE INDEX FK_MULE_EVENTS_BUSINESS_IDX ON MULE_EVENTS_BUSINESS (event_id);
 ----
 
-==== Internal Handler Configuration
+=== Internal Handler Configuration
 
-. Download the Microsoft JDBC driver from https://www.microsoft.com/en-us/download/details.aspx?displaylang=en&id=11774.
-. Extract the `sqljdbc_4_%version%.tar.gz` file to obtain the `sqljdbc4%version%_.jar` file.
-. Copy the JAR file to `{MULE_HOME}/plugins/mule-agent-plugin/lib/modules`.
+. Download the Microsoft JDBC driver for SQL Server from:
```
Comment (kbostrom) [inline modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc:290]:
```suggestion
. Download the Microsoft JDBC driver for SQL Server:
```
OR
```suggestion
. Download the Microsoft JDBC driver for SQL Server from `https://www.microsoft.com/en-us/download/details.aspx?displaylang=en&id=11774`.
```

File: modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc [hunk 8]
```diff
@@ -284,31 +285,31 @@ CREATE TABLE MULE_EVENTS_BUSINESS (
...
 == See Also
 
-* xref:monitoring.adoc[Monitoring Applications and Servers]
 * xref:sending-data-from-arm-to-external-analytics-software.adoc[Export Data to External Analytics Tools]
 * xref:managing-servers.adoc[Servers, Server Groups, and Clusters]
-* xref:deploying-to-your-own-servers.adoc[Deploying to Your Own Servers]
-* xref:managing-deployed-applications.adoc[Manage Deployed Applications]
-* xref:managing-applications-on-your-own-servers.adoc[Managing Applications on Your Own Servers]
-* https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/arm-rest-services[REST API]
+* xref:deploying-to-your-own-servers.adoc[Deploying to Your Servers]
```
Comment (kbostrom) [inline modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc]: ditto "Deploy"

File: modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc [hunk 9]
```diff
@@ -284,31 +285,31 @@ CREATE TABLE MULE_EVENTS_BUSINESS (
...
+* xref:managing-applications-on-your-own-servers.adoc[Managing Applications on Your Servers]
```
Comment (kbostrom) [inline modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc]: ditto "Manage"

**Review by kbostrom — APPROVED** [review]
Edited

**Pull request merged**